core/templates/pages/exploration-player-page/layout-directives/progress-nav.component.spec.ts

```diff
@@ -151,6 +155,22 @@ describe('Progress nav component', () => {
       mockDisplayedCard.getInteractionCustomizationArgs());
   }));
 
+  it('should respond to state card content updates', fakeAsync(() => {
+    let mockOnStateCardContentUpdate = new EventEmitter<void>();
+    spyOn(componentInstance, 'updateDisplayedCardInfo');
+    spyOnProperty(contentTranslationManagerService, 'onStateCardContentUpdate')
+      .and.returnValue(mockOnStateCardContentUpdate);
+
+    componentInstance.ngOnInit();
+    tick();
```

Comment:
why is there a `tick()` here?

Reply:
I added it for consistency with line 168 where I emit the specific event I am testing and then call tick. It may be that we add other events in the ngOnInit and triggering emitters for those should not result in a false positive.